.github/workflows/tests.yml

```diff
@@ -23,6 +21,19 @@ jobs:
         python-version: 3.8
     - uses: pre-commit/action@v2.0.0
 
+  pre-commit-hook:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Install pre-commit
+      run: pip install pre-commit
+    - name: run pre-commit
```

Comment:
```suggestion
    # Run pre-commit with try-repo to test the local version of the pre-commit hook
    # because we cannot initialize a local repo easily with the normal configuration file.
    # See https://pre-commit.com/#developing-hooks-interactively for details.
    - name: run pre-commit
```